### PR TITLE
Update servicenow plugin docs to include auto approval info

### PR DESCRIPTION
**Pull request by EdwardDowling**
Updates servicenow plugin docs to include info about the auto approval configuration

### CODE REVIEW

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 1]
```diff
@@ -9,7 +9,9 @@ that can become a vector for attacks.
 
 Teleport's ServiceNow integration allows you to treat Teleport Access
 Requests as ServiceNow incidents, notify the appropriate on-call team, and
-approve or deny the requests via Teleport.
+approve or deny the requests via Teleport. You can also configure the plugin to
+approve Access Requests automatically if the user making the request is in
+a specific on-call rotation
```
Comment (ptgott) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:14]:
```suggestion
a specific on-call rotation.
```

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 2]
```diff
@@ -35,6 +37,27 @@ API.
 Before making the access request ensure the user making the request has
 the `requester` role.
 
+For the plugin to know which serviceno rotations to check for the
+auto approval flow, the rota you wish users to be checked against
+must be included in a role the requesting user has. e.g.
```
Comment (ptgott) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:42]:
```suggestion
For the plugin to know which ServiceNow rotations to check for the
auto approval flow, the rotation you wish users to be checked against
must be included in one of the requesting user's roles. To do so, add the `teleport.dev/schedules` annotation as shown in the following example: 
```

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 3]
```diff
@@ -35,6 +37,27 @@ API.
...
+
+```
```
Comment (ptgott) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:44]:
```suggestion
```yaml
```

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 4]
```diff
@@ -56,6 +79,10 @@ As the Teleport user `myuser`, create an Access Request for the `editor` role:
 In ServiceNow, you will see a new incident containing information about the
 Access Request.
 
+If the a ServiceNow rotation was specified in the requesters role annotations,
```
Comment (ptgott) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:82]:
```suggestion
If the a ServiceNow rotation was specified in the requester's role annotations,
```

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 5]
```diff
@@ -56,6 +79,10 @@ As the Teleport user `myuser`, create an Access Request for the `editor` role:
...
+then the ServiceNow plugin will check that the requesters username matches the
```
Comment (ptgott) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:83]:
```suggestion
then the ServiceNow plugin will check that the requester's username matches the
```
We use apostrophes in possessive nouns

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 6]
```diff
@@ -56,6 +79,10 @@ As the Teleport user `myuser`, create an Access Request for the `editor` role:
...
+ServiceNow user currently on-call in that rotation and approve the access request.
```
Comment (ptgott) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:84]:
```suggestion
ServiceNow user currently on-call in that rotation and approve the Access Request.
```
For consistency with the rest of the docs.

**Comment by ptgott** [comment]
@EdwardDowling Are we backporting this, or is it behavior introduced in v15?

**Comment by EdwardDowling** [comment]
> @EdwardDowling Are we backporting this, or is it behavior introduced in v15?

We are backporting  it, will add the label now

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 7]
```diff
@@ -35,6 +37,27 @@ API.
...
+spec:
+  allow:
+    request:
+      roles: ['editor']
+      thresholds:
+        - approve: 1
+          deny: 1
+      annotations:
+        teleport.dev/schedules:
+        - YOUR_SERVICENOW_ROTA_ID_HERE
```
Comment (r0mant) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:58]: We should explain below how to determine the rotation ID.

Comment (EdwardDowling) [reply docs/pages/access-controls/access-request-plugins/servicenow.mdx:58]: Added the method with the fewest steps I could find.

**Review by ptgott — APPROVED** [review]

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 8]
```diff
@@ -35,6 +37,35 @@ API.
...
+```
+
+To retrieve the ServiceNow rotation ID, first navigate to the group record
+ of the ServiceNow group the rotation belongs to and right click on
+ header->'Select copy sys_id' to copy the sys_id.
```
Comment (zmb3) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:63]:
```suggestion
To retrieve the ServiceNow rotation ID, navigate to the group record
of the ServiceNow group the rotation belongs to and right click on
header, then click 'Select copy sys_id' to copy the ID.
```

File: docs/pages/access-controls/access-request-plugins/servicenow.mdx [hunk 9]
```diff
@@ -35,6 +37,35 @@ API.
...
+
+Then using the ServiceNow endpoint '/api/now/on_call_rota/workbench/group/{groupSysId}'
```
Comment (zmb3) [inline docs/pages/access-controls/access-request-plugins/servicenow.mdx:65]: This still feels hard to follow. How am I supposed to hit the endpoint? Do I need credentials?

Comment (EdwardDowling) [reply docs/pages/access-controls/access-request-plugins/servicenow.mdx:65]: Good point. Would an example curl request be better?

**Review by zmb3 — APPROVED** [review]
